build/yamls/patches/kind/startOvs.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apps/v1
```

Comment:
What is the naming convention for yaml? startOVS? start_ovs?

Reply:
I just followed the "convention" for the files I already had (camel case without capitalizing every letter in "ovs"). I already had `ovsDatapath.yml` so I went with `startOvs.yml`.